Tools/autotest/arduplane.py

```diff
@@ -7232,6 +7232,136 @@ def Volz(self):
             if abs(m.Pos - m.PosCmd) > 20:
                 break
 
+    def setup_servo_mount(self, roll_servo=5, pitch_servo=6, yaw_servo=7):
+        '''configure a rpy servo mount; caller responsible for required rebooting'''
+        self.progress("Setting up servo mount")
+        self.set_parameters({
+            "MNT1_TYPE": 1,
+            "MNT1_PITCH_MIN": -45,
+            "MNT1_PITCH_MAX": 45,
+            "RC6_OPTION": 213,  # MOUNT1_PITCH
+            "SERVO%u_FUNCTION" % roll_servo: 8, # roll
+            "SERVO%u_FUNCTION" % pitch_servo: 7, # pitch
+            "SERVO%u_FUNCTION" % yaw_servo: 6, # yaw
+        })
+
+    def ScriptedArmingChecksApplet(self):
+        """ Applet for Arming Checks will prevent a vehicle from arming based on scripted checks
+            """
+        self .start_subtest("Scripted Arming Checks Applet validation")
```

Comment:
```suggestion
        self.start_subtest("Scripted Arming Checks Applet validation")
```

A few places like this

Reply:
fixed